extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -233,6 +281,20 @@ const stateComparisonFunctions = {
  */
 const actions = (state, action, data) => {
   switch (action) {
+    case Action.ADD_INTERACTIVE_REACT:
+      state = /** @type {!State} */ ({
+        ...state,
+        [StateProperty.INTERACTIVE_REACT_STATE]: {
+          ...state[StateProperty.INTERACTIVE_REACT_STATE],
+          [data['interactiveId']]: data,
+        },
+      });
+      state[
+        StateProperty.INTERACTIVE_RESULTS_STATE
+      ] = processInteractiveResults(
+        state[StateProperty.INTERACTIVE_REACT_STATE]
+      );
```

Comment:
When you send an ADD_INTERACTIVE_REACT, it sets state[interactiveReactState][interactiveId] = data, and then calculates the results state and puts it into state.interactiveResultsState